guides/v2.2/cloud/live/live-sanity-check.md

```diff
@@ -187,7 +187,7 @@ We highly recommend having Magento already installed prior to deployment. During
     php bin/magento deploy:mode:set production
     ```
 
-If errors display, debug them if possible, [review logs]({{ page.baseurl }}/cloud/trouble/environments-logs.html), or open a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to get additional assistance.
+If errors display, debug them if possible, [review logs]({{page.baseurl}}/cloud/project/log-locations.html), or open a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to get additional assistance.
```

Comment:
```suggestion
If errors display, debug them if possible by [reviewing logs]({{page.baseurl}}/cloud/project/log-locations.html). Open a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to get additional assistance.
```

Reply:
Not sure the suggestion is the best, but the existing sentence could be improved.